Ssh/SshShell.php

```diff
@@ -8,6 +8,7 @@
 use phpseclib\Net\SSH2;
 use RuntimeException;
 
+// @codeCoverageIgnoreStart
```

Comment:
Inconsistency, here no empty line after the comment, at the codeCoverageIgnoreEnd comment there is a line before.